verif/big_core/tests/floating_point.c

```diff
@@ -1,10 +1,10 @@
-#include <stdio.h>
+#include "../../../app/defines/big_core_defines.h"
 
 int main() {
     float a = 1.5, b = 2.5;
```

Comment:
You didn't do any operation here..
Did you run out of memory?

Reply:
Still have to figure out how to cast the result from int to float.

Reply:
Don't worry about it.
Let's get other tests running with other features such as CR MMIO, and VGA.
Then we can re-visit this.